### ISSUE TITLE
[AVCaptureSession startRunning] startRunning crash 

### ISSUE THREAD

**Issue by badarovska** [opentok/opentok-ios-sdk-samples#279]
Sometimes when publishing a stream, I am getting the following error: 

**_[AVCaptureSession startRunning] startRunning may not be called between calls to beginConfiguration and commitConfiguration_**

I have not found specific steps to reproduce it. It is not often, but it is happening on my production app as well. In the live app I am using Opentok 2.24.1, but I upgraded to 2.24.2 locally and the issue remains. 

- iOS SDK version: Opentok 2.24.1, Opentok 2.24.2 
- OS and version: iOS 14, iOS 15, iOS 16

Logs from Firebase: 
<img width="606" alt="Screenshot 2023-03-24 at 10 14 38" src="https://user-images.githubusercontent.com/17143444/227476434-4ecaf13f-c232-4cd0-8bd4-588f43d027b2.png">

**Comment by v-kpheng** [comment]
Thanks, @badarovska, for letting us know about this issue.

We've been able to reproduce.  We're working on a fix now: https://jira.vonage.com/browse/VIDCS-13.

**Comment by v-kpheng** [comment]
Just wanted to share another update.  We hope this will be fixed in the upcoming 2.25.0 release.

**Comment by beHaze** [comment]
Fixed 2.25.0 release

**Comment by egarni** [comment]
Hello, I updated to 2.25.0 but I got now the following crash:
```
Fatal Exception: NSGenericException
0  CoreFoundation                 0x9e38 __exceptionPreprocess
1  libobjc.A.dylib                0x178d8 objc_exception_throw
2  AVFCapture                     0x3f918 -[AVCaptureSession stopRunning]
3  libdispatch.dylib              0x2460 _dispatch_call_block_and_release
4  libdispatch.dylib              0x3f88 _dispatch_client_callout
5  libdispatch.dylib              0x15b8c _dispatch_root_queue_drain
6  libdispatch.dylib              0x16284 _dispatch_worker_thread2
7  libsystem_pthread.dylib        0xdbc _pthread_wqthread
8  libsystem_pthread.dylib        0xb98 start_wqthread
```
